api/admin/controller/library_settings.py

```diff
@@ -68,12 +70,15 @@ def process_get(self):
             )]
         return dict(libraries=libraries, settings=Configuration.LIBRARY_SETTINGS)
 
-    def process_post(self, validator=None):
+    def process_post(self, validators_by_type=None):
```

Comment:
It would be even better if we could eventually fold the calls to the specialized validators into `validate_formats` (in `/api/admin/controller/__init__.py`), i.e. run all the validators from the same place instead of treating these like a whole different category of thing.  (But that may be a ways down the road--just something to think about in the meantime!)

Reply:
Absolutely, this is where I want to go as well.